### ISSUE TITLE
Chapman-Kolmogorov validation

### ISSUE THREAD

**Issue by jchodera** [choderalab/msm-pipeline#26]
We should be using the [Chapman-Kolmogorov validation](http://www.emma-project.org/latest/api/generated/pyemma.msm.ChapmanKolmogorovValidator.html#pyemma.msm.ChapmanKolmogorovValidator) on our final MSMs as another check of self-consistency.

**Comment by maxentile** [comment]
Good call! I had mentioned that in the readme but didn't include it yet. Will add few lines for this in my next pull request.

We'll need to specify m groups of states of interest here. Shall we just use whatever macrostate definitions we settle on here?

**Comment by jchodera** [comment]
It might be good to check out the best practices MSM review from Frank and
I to see what the specific recommendations is. The number of trajectories
cisiting individual microstates is usually so low that the correlation
functions are very noisy, so I believe they recommend using random
collections of macrostates whose total probability or counts are above some
threshold.

**Comment by maxentile** [comment]
Thanks, John!

Sorry, I'm having trouble finding your reference that recommends to use random collections of macrostates-- could you please point it out to me?
- In section IV.F. of [MSMs of molecular kinetics: generation and validation (2011)](http://www.diss.fu-berlin.de/docs/servlets/MCRFileNodeServlet/FUDOCS_derivate_000000005544/1.3565032.pdf) (and Chapter 4.8 of [the MSM book](http://www.springer.com/us/book/9789400776050)), you used the top 6 metastable states identified by PCCA.
- In [MSMs of biomolecular conformational dynamics (2014)](http://publications.mi.fu-berlin.de/1411/1/14_CurrOpin_ChoderaNoe_MSMReview.pdf), the CK test doesn't appear to be explicitly mentioned.
- In [Automatic discovery of metastable states (2007)](http://people.cs.uchicago.edu/~nshinrichs/NinaHinrichs-AutoStateDecomp-JCP-2007.pdf), 3 states were monitored in the CK test (Figure 6 + section IV.B.2), but I didn't see where the method used to select those states was described.

**Comment by jchodera** [comment]
> In section IV.F. of MSMs of molecular kinetics: generation and validation
> (2011)
> http://www.diss.fu-berlin.de/docs/servlets/MCRFileNodeServlet/FUDOCS_derivate_000000005544/1.3565032.pdf
>  (and Chapter 4.8 of the MSM book
> http://www.springer.com/us/book/9789400776050), you used the top 6
> metastable states identified by PCCA.

I think this was the review I was thinking of!
